R_D_HW_4/homework4.py

```diff
@@ -0,0 +1,8 @@
+text = input()
```

Comment:
It would be good to provide some description for users what you expect they should enter